### PR TITLE
Adding CLI support on Aruba 5400 switches

### PR DESCRIPTION
**Pull request by lzammit**
# Description
Adding CLI support on Aruba 5400 switches

# Impacts
Switches

# NEWS file entries
Adding CLI support on Aruba 5400 switches

### CODE REVIEW

File: lib/pf/Switch/Aruba/5400.pm [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+   return [$status, %$radius_reply_ref];
+}
+
+=head1 AUTHOR
+
+Inverse inc. <info@inverse.ca>
+
+=head1 COPYRIGHT
+
+Copyright (C) 2005-2017 Inverse inc.
```
Comment (extrafu) [inline lib/pf/Switch/Aruba/5400.pm]: Wrong year.

File: lib/pf/Switch/Aruba/5400.pm [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+
+=head1 BUGS AND LIMITATIONS
+
+=cut
+
+use strict;
+use warnings;
+use Net::SNMP;
+
+use base ('pf::Switch::HP::Procurve_2500');
```
Comment (julsemaan) [inline lib/pf/Switch/Aruba/5400.pm]: Instead:
`use base ('pf::Switch::HP::Procurve_5400');`

File: lib/pf/Switch/Aruba/5400.pm [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+    $PORT
+);
+use pf::Switch::constants;
+use pf::util;
+
+sub description { 'Aruba 5400 Switch' }
+
+# CAPABILITIES
+# access technology supported
+sub supportsWiredMacAuth { return $TRUE; }
```
Comment (julsemaan) [inline lib/pf/Switch/Aruba/5400.pm]: Remove this after fixing the use base

File: lib/pf/Switch/Aruba/5400.pm [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+sub supportsWiredDot1x { return $TRUE; }
```
Comment (julsemaan) [inline lib/pf/Switch/Aruba/5400.pm]: Remove this after fixing the use base

File: lib/pf/Switch/Aruba/5400.pm [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+# VoIP technology supported
+sub supportsRadiusVoip { return $TRUE; }
```
Comment (julsemaan) [inline lib/pf/Switch/Aruba/5400.pm]: Remove this after fixing the use base

File: lib/pf/Switch/Aruba/5400.pm [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+# inline capabilities
+sub inlineCapabilities { return ($MAC,$PORT); }
```
Comment (julsemaan) [inline lib/pf/Switch/Aruba/5400.pm]: Remove this after fixing the use base

File: lib/pf/Switch/Aruba/5400.pm [hunk 7]
```diff
@@ -0,0 +1,163 @@
...
+
+#Insert your voice vlan name, not the ID.
+our $VOICEVLANAME = "voip";
```
Comment (julsemaan) [inline lib/pf/Switch/Aruba/5400.pm]: Remove this after fixing the use base

File: lib/pf/Switch/Aruba/5400.pm [hunk 8]
```diff
@@ -0,0 +1,163 @@
...
+
+=item getVoipVSA
+
+Get Voice over IP RADIUS Vendor Specific Attribute (VSA).
+
+TODO: Use Egress-VLANID instead. See: http://wiki.freeradius.org/HP#RFC+4675+%28multiple+tagged%2Funtagged+VLAN%29+Assignment
+
+=cut
+
+sub getVoipVsa {
```
Comment (julsemaan) [inline lib/pf/Switch/Aruba/5400.pm]: Remove this after fixing the use base

File: lib/pf/Switch/Aruba/5400.pm [hunk 9]
```diff
@@ -0,0 +1,163 @@
...
+    return ('Egress-VLAN-Name' => "1".$VOICEVLANAME);
+}
+
+=item isVoIPEnabled
+
+Is VoIP enabled for this device
+
+=cut
+
+sub isVoIPEnabled {
```
Comment (julsemaan) [inline lib/pf/Switch/Aruba/5400.pm]: Remove this after fixing the use base

**Comment by extrafu** [comment]
@lzammit Please adjust so we merge this.

**Comment by extrafu** [comment]
@julsemaan If all is fine with the changes @lzammit did, please merge.

**Pull request merged**